projects/packages/sync/src/modules/class-posts.php

```diff
@@ -618,12 +618,10 @@ public function save_published( $new_status, $old_status, $post ) {
 	 * The 2nd request is to update post meta, which is not supported on WP REST API.
 	 * When syncing post data, we will include if this was a meta box update.
 	 *
-	 * @todo Implement nonce verification.
-	 *
 	 * @return boolean Whether this is a Gutenberg meta box update.
 	 */
-	public function is_gutenberg_meta_box_update() {
-		// phpcs:disable WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended
+	private function is_gutenberg_meta_box_update() {
+		// phpcs:disable WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended -- We are using $_POST['action'] to detect if this is a Gutenberg meta box update.
```

Comment:
I'd say more like this
```suggestion
		// phpcs:disable WordPress.Security.NonceVerification.Missing, WordPress.Security.NonceVerification.Recommended -- Nonce checking, if necessary, is the responsibility of the caller. We can't know here what the caller is going to do with the return value.
```
And possibly also put a note in the function's doc block about the caller being responsible for checking a nonce if necessary.

Reply:
hmm if this becomes a private method we can be more explicit here I think and mention that we are only using it to detect if this is a Gutenberg meta box update and set a flag before sending the corresponding action to wpcom?

Reply:
I updated it based on the above, however feel free to suggest and commit any alternative explanation you feel is better :)